solarkraft/src/fetcher/storage.ts

```diff
@@ -107,6 +127,42 @@ export function loadContractCallEntry(filename: string): ContractCallEntry {
     }
 }
 
+/**
+ * Load fetcher state from the storage.
+ * @param root the storage root directory
+ * @returns the loaded state
+ */
+export function loadFetcherState(home: string): FetcherState {
+    const filename = getFetcherStateFilename(home)
+    if (!existsSync(filename)) {
+        // just return an empty map
+        return {
+            heights: OrderedMap<string, any>()
```

Comment:
Why not

```suggestion
            heights: OrderedMap<string, number>()
```

Reply:
Now I am surprised that `tsc` has not reported that :)